CoreWidgetProvider/Helpers/GPUStats.cs

```diff
@@ -93,9 +101,18 @@ public void GetData()
 
             if (success)
             {
-                var sum = counters?.Sum(x => x.NextValue());
-                gpu.Usage = sum.GetValueOrDefault(0) / 100;
-                ChartHelper.AddNextChartValue(sum.GetValueOrDefault(0), gpu.GpuChartValues);
+                try
+                {
+                    var sum = counters?.Sum(x => x.NextValue());
+                    gpu.Usage = sum.GetValueOrDefault(0) / 100;
+                    ChartHelper.AddNextChartValue(sum.GetValueOrDefault(0), gpu.GpuChartValues);
+                }
+                catch (InvalidOperationException ex)
+                {
+                    Log.Logger()?.ReportWarn("GPUStats", "InvalidOperationException", ex);
+                    Log.Logger()?.ReportInfo("GPUStats", "Calling GetGPUPerfCounters again");
+                    GetGPUPerfCounters();
```

Comment:
This has the ability to exhaust the stack if GetGPUPerfCounters continues to fail.

Reply:
@dhoehna Will it? GetGPUPerfCounters isn't calling itself and neither is GetData.

Reply:
...I must be missing something.  The method is called `GetGpuPerfCounters`  In the catch of try/catch line 115 calls `GetGPUPerfCounters`

Reply:
NVM.  I was wrong.  I was missing something.  I thought the try/catch that this PR has is in GetGPUPerfCounter.